plugins/entando-plugin-jpkiebpm/src/main/java/org/entando/entando/plugins/jpkiebpm/aps/internalservlet/BpmCaseInstanceCommentsAction.java

```diff
@@ -41,15 +41,17 @@ public String view() {
             this.setFrontEndCaseData(frontEndCaseDataIn);
             String channelIn = extractWidgetConfig("channel");
             this.setChannel(channelIn);
-
             KieBpmConfig config = formManager.getKieServerConfigurations().get(this.getKnowledgeSourceId());
+            if (null == config) {
```

Comment:
Should this be in the base action? Seems repeated across all of the widgets. Maybe a call to the super class for the config to manage this error condition so we don't have the same block in every widget?